src/ro/siit/ObjectContainers/Main.java

```diff
@@ -0,0 +1,83 @@
+package ro.siit.ObjectContainers;
+
+import java.util.*;
+
+/**
+ * This code will store some persons in a TreeSet. After that, it will iterate through the TreeSet using 2 Comporators.
+ * For one person, it will display their hobbies using a HashMap.
+ * @author Catalina Neagu
+ */
+
+
+public class Main {
+    public static void main(String[] args) {
+        Person p1 = new Person("Ion", 10);
+        Person p2 = new Person("Andrei", 12);
+        Person p3 = new Person("Alexandra", 20);
+        Person p4 = new Person("Toma", 25);
+
+        //Iterate through the TreeSet using name comparator (String comparison)
+        TreeSet<Person> personComp = new TreeSet<Person>(new MyNameComp());
+        personComp.add(p1);
+        personComp.add(p2);
+        personComp.add(p3);
+        personComp.add(p4);
+        for (Person p : personComp) {
+            System.out.println(p);
+        }
+
+        //Iterate through the TreeSet using age comparator (int comparison)
+        TreeSet<Person> ageComp = new TreeSet<Person>(new MyAgeComp());
+        ageComp.add(p1);
+        ageComp.add(p2);
+        ageComp.add(p3);
+        ageComp.add(p4);
+        for (Person p : ageComp) {
+            System.out.println(p);
+        }
+
+        //Defining a HashMap
+        HashMap<Person, List<Hobby>> hobbyPerson = new HashMap<Person, List<Hobby>>();
+
+        //Defining 2 lists with addresses
+        ArrayList<String> addressList = new ArrayList<>();
+        addressList.add("Constanta");
+        addressList.add("Galati");
+
+        ArrayList<String> addressList2 = new ArrayList<>();
+        addressList2.add("Brasov");
+        addressList2.add("Timisoara");
+
+        //Defining a list of hobbies
+        List<Hobby> hobbiesList = new ArrayList<>();
+        Hobby h1 = new Hobby("swimming", 60, addressList);
+        Hobby h2 = new Hobby ("cycling", 3, addressList2);
+        hobbiesList.add(h1);
+        hobbiesList.add(h2);
+
+        hobbyPerson.put(p1, hobbiesList);
+        System.out.println(hobbyPerson);
```

Comment:
Almost there: the exercise however was saying "Add some information to this map; for a certain Persoana, print the names of the hobbies and the countries where it can be practiced" :)